pkg/file/pipeline/hashtrie/hashtrie.go

```diff
@@ -21,20 +25,29 @@ var (
 const maxLevel = 8
 
 type hashTrieWriter struct {
+	ctx                    context.Context // context for put function of dispersed replica chunks
```

Comment:
context should not be a field if possible

Reply:
hashtrie's Sum() method should satisfy pipeline interface that does not define context.
context is needed to put SOC replica.